.github/workflows/validate.yml

```diff
@@ -1,19 +1,32 @@
-name: Validate SDK
+name: Test
 
 on:
   schedule:
     - cron: "0 4 * * *"
   pull_request:
+  workflow_dispatch:
 
 jobs:
   deploy:
     name: Validate SDK
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [windows-2016, windows-2019, ubuntu-18.04, ubuntu-20.04]
+        php-version: [7.4, 8.0]
     steps:
     - name: Checkout
       uses: actions/checkout@v2
-    - name: Validate
-      uses: ./.github/actions/validate
+
+    - name: Setup PHP
+      uses: shivammathur/setup-php@v2
+      with:
+          php-version: ${{ matrix.php-version }}
+
+    - name: Install Packages
+      run: composer install
```

Comment:
does that setup install composer globally? and that is why you dont need `php composer.phar install`

Reply:
Yep - The action runner has php and composer installed globally with `composer` added to the path - so no need to install composer in the directory and run the file from the directory in GHA